jupyter-book/index.md

```diff
@@ -84,11 +84,15 @@ This is also published as a static website at:
 
   https://inria.github.io/scikit-learn-mooc/
 
+The videos are available as YouTube playlist at the Inria Learning Lab channel:
```

Comment:
I think that this should go below the information on the rocket icon, which I find quite important.